plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/glue/GlueIcebergTableOperationsProvider.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.iceberg.catalog.glue;
+
+import io.trino.plugin.hive.HdfsEnvironment.HdfsContext;
+import io.trino.plugin.iceberg.FileIoProvider;
+import io.trino.plugin.iceberg.catalog.IcebergTableOperations;
+import io.trino.plugin.iceberg.catalog.IcebergTableOperationsProvider;
+import io.trino.plugin.iceberg.catalog.TrinoCatalog;
+import io.trino.spi.connector.ConnectorSession;
+
+import javax.inject.Inject;
+
+import java.util.Optional;
+
+import static java.util.Objects.requireNonNull;
+
+public class GlueIcebergTableOperationsProvider
+        implements IcebergTableOperationsProvider
+{
+    private final FileIoProvider fileIoProvider;
+
+    @Inject
+    public GlueIcebergTableOperationsProvider(FileIoProvider fileIoProvider)
+    {
+        this.fileIoProvider = requireNonNull(fileIoProvider, "fileIoProvider is null");
+    }
+
+    @Override
+    public IcebergTableOperations createTableOperations(
+            TrinoCatalog catalog,
+            ConnectorSession session,
+            String database,
+            String table,
+            Optional<String> owner,
+            Optional<String> location)
+    {
+        TrinoGlueCatalog glueCatalog = (TrinoGlueCatalog) catalog;
+        return new GlueIcebergTableOperations(
+                glueCatalog.getGlueClient(),
+                glueCatalog.getStats(),
+                glueCatalog.getCatalogId(),
```

Comment:
Yeah, I agree. A wrapper is a bit of a pain just since the GlueClient API is so large.

What do you think about taking `catalogId` support out for now, fail if it is explicitly set, until we can add testing and a less error-prone approach?

Reply:
> What do you think about taking catalogId support out for now

i'm fine.

> fail if it is explicitly set

just remove config option (and have followup issue)

> until we can add testing and a less error-prone approach?

describe the "TAP" in the issue

Reply:
> just remove config option

The config options are being reused from the Hive connector's `GlueHiveMetastoreConfig` right now, so to do that I'd have to make a copy of the config class.

Reply:
oh, ok, fine. Let's have a check for now then